Admin list ✨ Add User List & User Details Pages with Search, Pagination, and Status Toggle

kelvinempires opened the pull request:
This PR implements the full User Management flow based on the provided Figma design. Key features include:

User List Page with:
  Search functionality
  Pagination for large datasets
   "View" button to navigate to individual user details
   "Activate" / "Deactivate" toggle for user status
 *User Details Page*
   Displays user info cleanly
   Fully responsive layout
🔧 Matches the exact styling and layout in the Figma design
 📱 Fully responsive across screen sizes

This update brings complete functionality to the user management module.

kelvinempires committed:
first commit

kelvinempires committed:
start working on admin list view

kelvinempires committed:
feat: add UserDetailPage component for displaying user information and task history

kelvinempires committed:
first commit

kelvinempires committed:
firts commit

kelvinempires committed:
user list structure

kelvinempires committed:
user details page implimet

kelvinempires committed:
Fix package-lock.json merge conflict and reinstall dependencies

ikemHood commented on .env.example:
```diff
@@ -1,22 +0,0 @@
-# Since the ".env" file is gitignored, you can use the ".env.example" file to
```
Please restore the .env.example file

ikemHood reviewed (changes requested):
Looks Good. 

Restore .env.example file 

Also, run `npm run format:write` to fix lint errors

kelvinempires committed:
feat: Implement user detail and list components with loading skeletons

- Added UserDetailComponent for displaying user details and tasks.
- Created UserDetailPageWrapper to encapsulate UserDetailComponent.
- Introduced UserDetailSkeleton for loading state representation.
- Updated loading.tsx to use UserDetailSkeleton.
- Refactored page.tsx to utilize UserDetailPageWrapper with suspense.
- Developed UserListComponent for displaying a list of users with pagination and search.
- Added UserListSkeleton for loading state in user list.
- Refactored users page.tsx to use UserListComponent with suspense.

ekumamatthew commented:
that is a table you have there... use the table..
